modin/config/envvars.py

```diff
@@ -155,6 +155,14 @@ def _get_default(cls):
         return CpuCount.get()
 
 
+class RayPartitionThreshold(EnvironmentVariable, type=int):
```

Comment:
This is quite specific, do users actually need such low level control over the storage semantics of this specific engine?

Reply:
I think, this may be useful for those who do not want to add additional lines with `import ray` and `ray init(...)` to their workflow/scrpipts. The whole contol is left to Modin, but not to the developer.

Reply:
Let's postpone this specific update to a new issue and PR. I'm not sure it is directly related to the partition API. It has to do with initialization of Ray.

Reply:
Okay, it makes sense.